proposals/evaluation/uc-3-inheritance.md

```diff
@@ -0,0 +1,125 @@
+# Access Inheritance Use Cases
+
+Within this document, the following namespace prefix bindings are used:
+
+| Prefix    | Namespace                           |
+| --------- | ----------------------------------- |
+| `acl`     | `http://www.w3.org/ns/auth/acl#`    |
+| `acp`     | `http://www.w3.org/ns/solid/acp#`   |
+| `ex`      | `https://example.com/`              |
+| `ldp`     | `http://www.w3.org/ns/ldp#`         |
+| `vcard`   | `http://www.w3.org/2006/vcard/ns#`  |
+
+## Read access on a collection of resources
+
+Alice can read all resources in a collection.
+
+### ACP
+
+```turtle
+ex:Collection1
+  acp:accessControl ex:AccessControl1 ;
+  ldp:contains ex:Resource1 .
+
+ex:AccessControl1
+  acp:applyMembers ex:Policy1 .
+
+ex:Policy1
+  acp:anyOf ex:AgentMatcher1 ;
+  acp:allow acl:Read .
+
+ex:AgentMatcher1
+  acp:agent ex:Alice .
+```
+
+Note: Resources linked to a collection via `ldp:contains` will inherit policies linked to the collection's access control via `acp:applyMembers`. Collections linked to a collection via `ldp:contains` will pass the inherited policies through to their member resources.
+
+## Read access to a group on a collection of resources
+
+See also: https://solid.github.io/authorization-panel/authorization-ucr/#inheritance-readonly
+
+### Setup
+
+The Weekly status collection is an `ldp:BasicContainer`, which contains a number of `ldp:BasicContainers`, one for each weekly meeting. The advantage of having these as containers rather than plain resources is that any number of other documents can be added to the container too.
+
+```turtle
+# Resource: </weekly-status/>
+<.>
+  a ldp:BasicContainer ;
+  ldp:contains <2021-04-28/>, <2021-05-05/>, <2021-05-12/> .
+```
+
+The `</weekly-status/.acl>` resource is advertised as `</weekly-status/>`'s access control list via a `Link` header with a relationship type of `http://www.w3.org/ns/auth/acl#accessControl`.
```

Comment:
Redundancy; line 68 repeats line 52. One should be deleted.
```suggestion
```

Reply:
Commited the other one.